packages/components/src/components/accordion/component.tsx

```diff
@@ -118,12 +132,22 @@ export class KolAccordion implements AccordionAPI {
 	}
 
 	public componentWillLoad(): void {
+		this.validateDisabled(this._disabled);
 		this.validateLabel(this._label);
 		this.validateLevel(this._level);
 		this.validateOn(this._on);
 		this.validateOpen(this._open);
 	}
 
+	public componentDidRender() {
+		if (this.host && this.host.shadowRoot) {
+			Array.from(this.host.shadowRoot.querySelectorAll('[aria-disabled="true"]')).forEach((el) => {
+				el.removeEventListener('click', preventDefault);
```

Comment:
Warum handlen wir das nicht in der bestehenden `onClick` Methode?

Reply:
Auch bei den anderen Komponenten, wäre ein einfaches `onClick` nicht robuster und einfacher nachzuvollziehen?

Reply:
Es stimmt, dass wir das Aufklappen über den Callback verhindern können. Dem Nutzenden müsste man dann in dieser Situation eine Meldung anzeigen. Möglich, aber sicher so aus UX eher nicht gewünscht. 

Das disabled würde des HTML gleich korrekt stylen und barrierefrei auszeichnen.

Reply:
@sdvg: Ich habe das Handling wieder entfernt. I.d.R. nutze ich den Click-Handler. Nur beim Details muss es am `summary`-Tag sein, weil sonst der Toggle schon erfolgt ist.

Reply:
@deleonio Wäre hier nicht trotzdem ein `<summary onClick=...>` möglich? Einfach, um näher an den Stencil-Konventionen zu bleiben?